pkg/api/aim/query/query.go

```diff
@@ -17,9 +17,11 @@ import (
 )
 
 const (
-	OperationEndsWith   = "endswith"
-	OperationContains   = "contains"
-	OperationStartsWith = "startswith"
+	RegexpOperationMatch  = "match"
+	RegexpOperationSearch = "search"
+	OperationEndsWith     = "endswith"
+	OperationContains     = "contains"
+	OperationStartsWith   = "startswith"
```

Comment:
actual question: do we need constants if they are used in one place only? (I'm curious, I don't have a strong opinion either way)

Reply:
not a big deal. we ca remove them. usually `constant` give you a more meaning but lets skip and see.